source/markdown/details/item-5.md

```diff
@@ -1,4 +1,4 @@
-For answers to questions not listed here, please [visit our Discourse forum](https://forum.learning.mozilla.org/c/nsf-wins).
+For answers to questions not listed here, please [visit our Discourse forum](https://forum.learning.mozilla.org/c/nsf-wins) or email [wirelesschallenge@mozillafoundation.org](wirelesschallenge@mozillafoundation.org).
```

Comment:
@mehan one markdown tip: we can include `mailto:` in markdown to make it a "mailto" link

```markdown

[wirelesschallenge@mozillafoundation.org](mailto:wirelesschallenge@mozillafoundation.org)

```

I can fix this line in a separate PR. 😄